src/style.css

```diff
@@ -1,3 +1,79 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  margin: 0;
+
+  --hover-color: #00acdc;
+}
+
+.header {
+  background-color: #fff;
+
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin: auto;
+}
+
+.logo {
+  display: inline-flex;
+  margin-left: 50px;
+}
+
+.logo_header {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_header {
+  margin-right: 50px;
+}
+
+.header_nav_list {
+  list-style: none;
   margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+}
+
+.nav_link {
+  height: 100%;
+  line-height: 60px;
+  display: block;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+}
+
+.header_nav_item:nth-child(n + 2) {
+  margin-left: 20px;
+}
+
+.nav_link:hover {
+  color: var(--hover-color);
+}
+
+.nav_link:active {
+  content: '';
+}
+
+.is-active {
+  color: var(--hover-color);
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  top: 56px;
+  display: block;
+  background-color: var(--hover-color);
+  border-radius: 8px;
+  height: 4px;
+  width: 37px;
```

Comment:
To not add specific px width, you can set `position: relative` for `.is-active` element and add` width: 100% `for `after` element

Also better to add `bottom: 0` instead of setting top